docs/notebooks/Poincare Evaluation.ipynb

```diff
@@ -1340,106 +1240,6 @@
        "        <th>100</th>\n",
        "        <th>200</th>\n",
        "    </tr>\n",
-       "    <tr>\n",
```

Comment:
Why this removed (and exactly same question for similar blocks after).

Reply:
Again this wasnt removed intentionally... the model was taking too long to run on my local system. I ended up just test running some of these cells(they are all working) and interrupting between. So the output is not present.

Reply:
same comment as before https://github.com/RaRe-Technologies/gensim/pull/1893#discussion_r167535243